articles/precarious-data.md

```diff
@@ -1,51 +1,217 @@
 
-Centralization is a disease that the web is suffering from. Information can exist in many places at once, yet we have built the web using links which pretend there's only one location where you can find the information they're pointing to. This makes locations artificially important, turning them into points of failure and putting great power and responsibility on whoever controls those locations. That's centralization: Despite the fact that information can exist in many places at once, we link to information as if it existed at singular locations, which forces us to rely on central locations whenever we want to acquire information or traverse links between pages, datasets, documents, etc. 
+The remedy for this centralization is to change how we link to content on the web. Information on the web is, by its nature, decentralized. We just pretend that it exists in discrete, centralized locations. The web itself is already decentralized; information already propagates across the web whenever people use it. We just need to start using that decentralized nature of information for our benefit rather than pretending that it's not happening.
 
+Decentrailzed web technologies will smooth the process of saving endangered data now and will prevent this problem from happening again in the future. Organizations like the Internet Archive, the Sloan Foundation and the Knight Foundation have been encouraging the use of decentralized technologies for some time. The most visible manifestation of this effort was the [Decentralized Web Summit](https://www.decentralizedweb.net) in June 2016.
 
-This centralization is a disease which has many symptoms. When someone shuts down a property on the web or decides to pervert the information at that location it can harm whole groups of people - industries, fields of research, global communities, etc. Sometimes in the face of a pending shutdown people scramble to save endangered data. Other times locations become unavailable suddenly due to attacks, neglect or pure coincidence and people have to cope with the data loss. Those are just two symptoms of centralization. There are many more.
+Technologies and protocols like [IPFS](https://ipfs.io) and [Dat](http://datproject.org) let you link to the content itself, wherever it is, based on its fingerprint. These free open source tools, built on proven techniques, offer the most direct path to decentralizing your data while retaining the integrity of those data. Even better, if there are already many copies of the data around the world, these tools will let you transform those many copies of your data into a resilient decentralized network of data providers that grows whenever someone reads the data or publishes a copy of the data they already had.  
```

Comment:
I don't know what authority you're appealing to with 'proven'. Might be good to specify, or it sounds a bit markety.